src/Backend/Modules/MediaLibrary/Installer/Installer.php

```diff
@@ -36,18 +37,14 @@ public function install()
     protected function addGitIgnoreFile()
     {
         $fs = new Filesystem();
-
-        // Save Dropdowns
         $fs->dumpFile(
             FRONTEND_FILES_PATH . '/MediaLibrary/.gitignore',
             '# Uploaded files - Do not delete anything in this directory
-            /Source/*
-            
-            # Generated images needed for backend - Do not delete anything in this directory
-            /Backend/*
-            
-            # Automagically generated images needed for frontend - All directories and images in this directory can safely be removed.
-            /Frontend/*'
+            *'
+        );
+        $fs->dumpFile(
+            FRONTEND_FILES_PATH . '/Cache/.gitignore',
+            '*'
```

Comment:
I wouldn't use php to generate this but add a file to the folder and exclude the .gitignore so it is inside the git repo

also I would use a more descriptive name than cache, cache can be a lot of things;

It is the image cache

this should also be cleared when de cache clear commando's are called